javascript/javascript2/week1/spirit-animal.js

```diff
@@ -0,0 +1,50 @@
+const getAnimalNameBtn = document.getElementById("getAnimalNameBtn");
+const animalName = document.getElementById("animal-name");
+const spiritAnimalList = [
+	"The fullmoon wolf",
+	"The Butterfly",
+	"The Spider",
+	"The Crow",
+	"The Owl",
+	"The Hummingbird",
+	"The Hawk",
+	"The Grasshopper",
+	"The Eagle",
+	"The crying butterfly",
+];
+function getAnimalName() {
+	const nameVal = animalName.value;
+	const newSpiritAnimal =
+		spiritAnimalList[Math.floor(Math.random() * spiritAnimalList.length)];
+	const displayNameResult = document.getElementById("displayNameResult");
+	if (nameVal) {
+		const output = `${nameVal} - ${newSpiritAnimal}`;
+		displayNameResult.innerText = output;
+	} else {
+		displayNameResult.innerText = "Please Enter Name";
+	}
+}
+
+const radioButtons = document.getElementsByName("select-type");
+
+for (const radioButton of radioButtons) {
+	radioButton.addEventListener("change", function () {
+		const value = radioButton.value;
+
+		if (value === "click") {
```

Comment:
Consider using a switch block instead - see here: https://www.w3schools.com/js/js_switch.asp

Reply:
okay. i will try with switch

Reply:
I have updated the code by using switch. Please check and thank you so much for your review.